_pages/get_involved.md

```diff
@@ -75,7 +75,7 @@ will have to wait for BIDS 2.0 are listed
 | [BEP022](https://bids.neuroimaging.io/bep022){:target:"_blank"} | Magnetic Resonance Spectroscopy (MRS)                                                                      | Mark Mikkelsen and William Clarke                                        | [Group discussions](https://forum.mrshub.org/t/bids-for-spectroscopy/83){:target:"_blank"} underway                                                                                                                                                                  |                                                                                        |
 | [BEP023](https://bids.neuroimaging.io/bep023){:target:"_blank"} | PET Preprocessing derivatives                                                                              | Martin Noergaard, Graham Searle, Melanie Ganz                 | Work is paused until BEP009 manuscript has been submitted/accepted.                                                                                                                                                                       | A blocking item is the need for more contributors, especially from senior PET experts. |
 | [BEP024](https://bids.neuroimaging.io/bep024){:target:"_blank"} | Computed Tomography scan (CT)                                                                              | Hugo Boniface                                                 | Lead seeking more contributors and experts.                                                                                                                                                                    | None.                                                                                  |
-| [BEP025](https://bids.neuroimaging.io/bep025){:target:"_blank"} | Medical Population Imaging Data structure (MIDS)                                                           | Jose Manuel Saborit Torres, Maria de la Iglesia Vayá          | Preparing to submit the preprint. Recently [published a preprint](https://arxiv.org/abs/2006.01174){:target:"_blank"} describing a use case of this BEP.                                                                                                                | None.                                                                                  |
+| [BEP025](https://bids.neuroimaging.io/bep025){:target:"_blank"} | Medical Imaging Data structure (MIDS)                                                                      | Jose Manuel Saborit Torres, Maria de la Iglesia Vayá          | BEP update. Preparing to submit the preprint. Recently [published a preprint](https://arxiv.org/abs/2006.01174){:target:"_blank"} describing a use case of this BEP.                                                                                                                | None.                                                                                  |
```

Comment:
I see two changes here:

1. `Medical Population Imaging Data structure` ... delete `Population`
1. Adding the words `BEP update.`

I see why you want to changed 1. --- but what are the changes under point 2. about?

Reply:
We wanted to point out that the BEP had been updated

Reply:
Ah I see, I would suggest that you remove it again, because it's clear from the table column header that this is a BEP update.

(That is, we don't document that we "update an update")

Reply:
Ok, we are remove it